shared/studio/tabs/auth/providers.tsx

```diff
@@ -50,11 +51,28 @@ export const ProvidersTab = observer(function ProvidersTab() {
       {state.providers ? (
         <>
           {state.providers.length ? (
-            <div className={styles.cardList}>
-              {state.providers.map((provider) => (
-                <ProviderCard key={provider.name} provider={provider} />
-              ))}
-            </div>
+            <>
+              {state.noEmailProviderWarning ? (
+                <div className={styles.emailProviderWarning}>
+                  <WarningIcon />
+                  <span>Warning:</span> You have enabled auth providers
+                  requiring email, but no SMTP provider is configured.
```

Comment:
It's a bit more complicated than this since you might avoid configuring SMTP and just use webhooks.

Reply:
Now that I think of it, we should have an opt-out option which sets the pointer to `{}` without losing your config.

Reply:
Ok, I've updated the warnings to be more specific and also take into account webhook config. And added a button deselect the currently enabled smtp provider.